<launch>
    <node
        pkg="spot_person_leader"
        type="lead_person_demo.py"
        name="spot_person_leader"
        output="screen"
        >
        <!-- Action Client -->
        <remap from="~lead_person" to="/spot/lead_person" />

        <rosparam file="$(find spot_person_leader)/config/map.yaml" />
    </node>
</launch>
